Dockerfile

```diff
@@ -37,7 +37,6 @@ COPY \
 COPY app app
 COPY config config
 COPY scripts scripts
-COPY assets assets
 COPY styleguide styleguide
```

Comment:
Can we not change this to `COPY app/assets assets` ?

Reply:
Isn't this handled with COPY app app

Reply:
That would require lots of webpack includes paths changed too but yes we can if theres a benefit to these existing separately on the server.